<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsDialogClass</class>
 <widget class="QDialog" name="SettingsDialogClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>564</width>
    <height>451</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SettingsDialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QListWidget" name="listWidget">
       <property name="maximumSize">
        <size>
         <width>150</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QStackedWidget" name="stackedWidget">
       <property name="currentIndex">
        <number>1</number>
       </property>
       <widget class="QWidget" name="accountsPage">
        <widget class="QPushButton" name="pushButton">
         <property name="geometry">
          <rect>
           <x>230</x>
           <y>50</y>
           <width>78</width>
           <height>25</height>
          </rect>
         </property>
         <property name="text">
          <string>add</string>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="pluginsPage">
        <widget class="QListView" name="listView">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>60</y>
           <width>301</width>
           <height>261</height>
          </rect>
         </property>
        </widget>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="serviceSettingsPane" native="true"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
